examples/experimental/dagster-components/dagster_components/impls/sling_replication.py

```diff
@@ -0,0 +1,44 @@
+import os
+from pathlib import Path
+
+import yaml
+from dagster._core.definitions.definitions_class import Definitions
+from dagster_embedded_elt.sling import SlingResource, sling_assets
+from dagster_embedded_elt.sling.resources import AssetExecutionContext
+from pydantic import TypeAdapter
+from typing_extensions import Self
+
+from dagster_components import Component, ComponentLoadContext
+from dagster_components.core.component_decl_builder import ComponentDeclNode, YamlComponentDecl
+
+
+class SlingReplicationComponent(Component):
+    params_schema = SlingResource
```

Comment:
note: we can just use a configurable resource as the params schema, which is nice

Reply:
Not sure we want to normalize this given the complexity of partial resources and the like.